docs/gadget-devel/gadget-wasm-api-raw.md

```diff
@@ -361,15 +361,16 @@ params:
      ...
 ```
 
-#### `getParamValue(key string) string`
+#### `getParamValue(key string, dst uint64) uint64`
```

Comment:
Why not returning uint32? Some other functions that return "0 in case of success, 1 otherwise" use a uint32.

Reply:
Same as above.